Gemfile

```
@@ -36,13 +36,15 @@ gem "bootsnap", require: false
 
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
-  gem "debug", platforms: %i[ mri windows ], require: "debug/prelude"
+  gem "debug", "~> 1.10.0", platforms: %i[mri windows], require: "debug/prelude"
```

```suggestion
  gem "debug", platforms: %i[mri windows], require: "debug/prelude"
```

As per above comment